front-end/src/components/AboutSurgeBanner/index.tsx

```diff
@@ -60,7 +61,7 @@ const styles = StyleSheet.create({
 
 export default function AboutSurgeBanner(): JSX.Element {
   return (
-    <div className={css(styles.wrapper)} id={STRINGS.aboutSurge}>
+      <div className={css(styles.wrapper)} id={STRINGS.aboutSurge}>
```

Comment:
Please fix indentation here. The `div` below should be inside this one, not at the same level.